src/components/token/Tokens.js

```diff
@@ -20,20 +23,155 @@ class Tokens extends React.Component {
     constructor(props) {
         super(props);
 
+        /**
+         * tokens: List of tokens currently being rendered
+         * hasAfter: Indicates if a next page exists
+         * hasBefore: Indicates if a previous page exists
+         * searchText: Input text written by user
+         * sortBy: Which field to sort (uid, name, symbol)
+         * order: If sorted field must be ordered asc or desc
+         * page: Current page. Used to know if there is a previous page
+         * pageSearchAfter: searchAfter param needed to get the next page
+         * loading: Initial loading, when user clicks on the Tokens navigation item
+         * isSearchLoading: Indicates if search results are being retrieved from explorer-service
+         * calculatingPage: Indicates if next page is being retrieved from explorer-service
+         */
         this.state = {
             tokens: [],
             hasAfter: false,
             hasBefore: false,
-            searchText: ""
+            searchText: "",
+            sortBy: "uid",
```

Comment:
Maybe sort by name, symbol or timestamp by default

Reply:
We will sort by timestamp, but we will need to change Logstash, explorer-service and explorer. This will be too much for this PR and I will send a new one with this change. Does that sound good to you?